pkgs/by-name/tt/ttaenc/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "ttaenc";
+  version = "3.4.1";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/tta/${finalAttrs.name}-${finalAttrs.version}-src.tgz";
```

Comment:
I think there is a sourceforge fetcher, please check whether you can use it.

Reply:
Didn't find one, but I saw other packages use `mirror://sourceforge` for mirror so I updated accordingly.

Reply:
Oh right, yes. That's what I meant :)